src/ui/pages/Connections/Connections.tsx

```diff
@@ -199,6 +212,28 @@ const Connections = ({
 
   useSwipeBack(getConnectionsTab, canStart, () => setShowConnections(false));
 
+  const deletePendingCheckProps = useMemo(
+    () => ({
+      title: i18n.t("connections.tab.detelepending.title"),
+      description: i18n.t("connections.tab.detelepending.description"),
+      button: i18n.t("connections.tab.detelepending.button"),
+    }),
+    []
+  );
+
+  const deleteConnection = async () => {
+    if (!deletePendingItem) return;
+
+    try {
+      setDeletePendingItem(null);
+      await Agent.agent.connections.deleteConnectionById(deletePendingItem.id);
+      dispatch(setToastMsg(ToastMsgType.CONNECTION_DELETED));
+      dispatch(removeConnectionCache(deletePendingItem.id));
+    } catch (error) {
+      // TODO
```

Comment:
@sdisalvo-crd @Sotatek-DukeVu If possible, can we from now one at least logging the exception if it's not meant to happen. Empty catches like that should be no where in the code. If we catch and show the toast, great, but lets also log it for development because there's been many times where I haven't known why something went wrong and had to dig into the code to find the empty catch block.

The only time you wouldn't need to log it is if you are catching a specific error message that you know can happen easily based on the users interactions. In this case, the delete should not fail unless there's a bug

Reply:
Great, thanks @iFergal so in this case both toast message and throw error in the console 👍

Reply:
I should plan a ticket for handling a full review of all the TODOs in our project at some point.

Reply:
I created it a task recently for that - https://cardanofoundation.atlassian.net/browse/DTIS-1162

Reply:
Thanks, I forgot about that one so I have created [another one](https://cardanofoundation.atlassian.net/issues/DTIS-1199) 😂 

I'll leave mine in my board and link it to yours as it will implement just that, so you will have yours for tracking purposes.

Reply:
They are duplicates though, you can just steal mine if you want as it has more detailed info